lutris/gui/lutriswindow.py

```diff
@@ -439,8 +443,11 @@ def _open_browser(url):
 
     @GtkTemplate.Callback
     def on_resize(self, widget, *args):
-        """WTF is this doing?"""
-        self.window_size = widget.get_size()
+        """ size-allocate signal - updates stored window size and maximized state """
```

Comment:
This is a better description of what the method does, but the formatting is worse. 🙈

Check out https://www.python.org/dev/peps/pep-0008/#documentation-strings / https://www.python.org/dev/peps/pep-0257/